src/views/screens/guided_search/geography_questionnaire.njk

```diff
@@ -6,6 +6,12 @@
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
+{% block head %}
+  {{ super() }}
+  <link rel="stylesheet" href="{{ assetPath }}/css/ol.css">
+  <script src="{{ assetPath }}/scripts/ol.js"></script>
```

Comment:
do you want this script to go into the header? can it not be at the end of body?

Reply:
Yes, we can place this at the end of the body tag. Now it have been moved. The reason for placing this in the head is, it was mentioned in the openlayer instructions.